### DISCUSSION TITLE
习题中的教材习题

### DISCUSSION THREAD

**Discussion by xcz0** [yhwu-is/Linear-Algebra-I-H-for-ZJU-CKC#60] [General]
可否将习题中提到的教材习题贴到对应位置，方便读者单独阅读此材料。

**Comment by shadowash0215** [comment]
直接在里面贴教材习题的图片的话，感觉排版上会不太好看，并且本书目前的面向读者还是以这两本书为教材的学生，参考上还是较为方便，电子版的书籍也可以从网上获取，所以这个还是先不考虑
